griptape/structures/structure.py

```diff
@@ -163,6 +165,11 @@ def before_run(self, args: Any) -> None:
 
     @observable
     def after_run(self) -> None:
+        if self.conversation_memory and self.output_task.output is not None:
```

Comment:
do you need the second `is not None` check now?

Reply:
There were some tests that relied on `output_task.output` being `None` which is still a valid state. Didn't feel like refactoring those.